app/decorators/article_decorator.rb

```diff
@@ -99,7 +99,7 @@ def co_authors
 
   def co_author_name_and_path
     co_authors.map do |user|
-      "<b><a href=\"#{user.path}\">#{user.name}</a></b>"
+      "<a href=\"#{user.path}\" class=\"crayons-link\">#{user.name}</a>"
```

Comment:
Ruby  has another type of string literal that allows interpolation and does not need escaping of double-quotes, let's use that here:

```suggestion
      %{<a href="#{user.path}" class="crayons-link">#{user.name}</a>}
```

Reply:
Oh, this looks so much better, thanks!